### PR TITLE
Scheduler tests

### PR DESCRIPTION
**Pull request by gluke77**

### CODE REVIEW

File: scheduler/scheduler_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	}
+
+	depGraph := *depGraphPtr
+
+	if len(depGraph) != 2 {
+		t.Errorf("Wrong length of dependency graph, expected %d, actual %d",
+			2, len(depGraph))
+	}
+
+	sr, ok := (depGraph)["pod/ready-1"]
```
Comment (nebril) [inline scheduler/scheduler_test.go]: Why the parentheses?

**Comment by nebril** [comment]
I think that IsBlocked should also be tested. Otherwise looks great!

**Comment by gluke77** [comment]
@nebril updated -- added test for IsBlocked()

**Comment by gluke77** [comment]
partially fixes #30

**Pull request merged**